rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py

```diff
@@ -48,13 +48,11 @@ def _get_doc(self, message: Message, attribute: Text) -> Any:
         """
         return message.get(LANGUAGE_MODEL_DOCS[attribute])
 
-    def process(self, message: Message, attribute: Text = TEXT, **kwargs: Any) -> None:
+    def process(self, message: Message, **kwargs: Any) -> None:
         """Sets the dense features from the language model doc to the incoming
         message."""
-        if attribute not in DENSE_FEATURIZABLE_ATTRIBUTES:
-            return
-
-        self._set_lm_features(message, attribute)
+        for attribute in DENSE_FEATURIZABLE_ATTRIBUTES:
```

Comment:
do we need to check `if message.get(attribute)`?

Reply:
No, because it is done in `_set_lm_features`

Reply:
I was wondering because the `convert_featurizer` does it as part of `process` 👍